apps/token-manager/app/src/screens/Holders.js

```diff
@@ -64,15 +99,125 @@ class Holders extends React.Component {
       </TwoPanels>
     )
   }
+
+  handleSelectTab = index => {
+    this.setState({ selectedTab: index })
+  }
+}
+
+const Screen = ({
+  offset,
+  children,
+  Component,
+  selected,
+  ...props
+}) => {
+  return (
+    <Spring
+      from={{ progress: 0 }}
+      to={{ progress: !!selected }}
+      config={springs.clamp}
+      native
+    >
+      {({ progress }) =>
+        selected && (
+          <AnimatedDiv
+            style={{
+              opacity: progress,
+              transform: progress.interpolate(
+                v => `translate3d(${offset - v * offset}px, 0, 0)`
+              ),
+            }}
+          >
+            <Component {...props} children={children} />
+          </AnimatedDiv>
+        )
+      }
+    </Spring>
+  )
 }
 
+const AnimatedDiv = styled(animated.div)`
+  position: relative;
+`
+
+const StyledTableHeader = styled(TableHeader)`
+  width: ${({ groupmode }) => (groupmode ? 100 : 50)}%;
+
+  ${breakpoint(
+    'medium',
+    `
+      width: auto;
+    `
+  )};
+`
+
+const ResponsiveTabBar = styled.div`
+  margin-top: 16px;
+
+  & ul {
+    border-bottom: none !important;
+  }
+  & li {
+    padding: 0 20px;
+  }
+
+  ${breakpoint('medium', `display: none`)};
```

Comment:
Rather than hiding this with css, could we just avoid rendering this component if `tabbedNavigation` is falsey?